module-workers.tf

```diff
@@ -36,7 +36,7 @@ module "workers" {
   cluster_dns            = var.cluster_dns
   cluster_id             = coalesce(var.cluster_id, one(module.cluster[*].cluster_id))
   cluster_type           = var.cluster_type
-  kubernetes_version     = var.kubernetes_version
+  kubernetes_version     = coalesce(one(module.cluster[*].cp_kubernetes_version), var.kubernetes_version)
```

Comment:
It is a bit long, but we updated a number of other variable names to align on the `control_plane_` prefix for clarity. That said, I also think we can leave this output unchanged as it's already scoped to the cluster submodule, and still enable overrides with the same `kubernetes_version` key optionally specified on entries of `worker_pools`.

Reply:
I've updated the logic and the output from the cluster submodule is no longer required.